x-pack/plugins/synthetics/public/apps/synthetics/components/monitor_add_edit/form/run_test_btn.tsx

```diff
@@ -125,7 +127,7 @@ export const TEST_SCHEDULED_LABEL = i18n.translate(
   }
 );
 
-const PRIVATE_AVAILABLE_LABEL = i18n.translate(
+export const PRIVATE_AVAILABLE_LABEL = i18n.translate(
   'xpack.synthetics.monitorList.testNow.available.private',
   {
     defaultMessage: `You can't currently test monitors running on private locations on demand.`,
```

Comment:
```suggestion
    defaultMessage: `You can't run tests on a private location.`,
```
Hi, suggesting a simpler edit, assuming that this is an action possible only on a single monitor instance. Is there something users can do to change this? If so, we could explain it in a few words. Otherwise, I don't think that "currently" and "on-demand" are needed considering the context.

Reply:
Actually this is a manual test run on demand, this doesn't imply that they can't run tests at all on private locations.

Reply:
Yes, I realized that my 1st suggestion was confusing and I updated it. How does it sound?